system_design/design_web_crawler.md

```diff
@@ -0,0 +1,228 @@
+
+# Web Crawler
+
+**Uses**
+
+Web crawlers are useful for accomplishing the following tasks.
+- How search engines get their data (ex. Google, Baidu)
+- Test web page links
+- Monitor content or structure updates on a page
+- Mirroring 
+    - mirroring: making a copy of a website
+- Copyright infrigement checks
+
+This specific solution does not attempt to address both the crawler AND a use case above. It is only attempting to address the crawler design.
+
+**Design Specifications Needed to Pass this Interview**
+Here are the specific sections you will need to address in a web crawler system design interview.
```

Comment:
Add space.

Reply:
@rogue0137 These two lines feel like they need s line break, but when I preview in markdown they're on the same line.